esp/src/src/ESPRequest.ts

```diff
@@ -189,12 +193,16 @@ class RequestHelper {
                                 }
                                 break;
                         }
+                        message = response.Exceptions.Exception[0].Message;
                     }
                     topic.publish("hpcc/brToaster", {
                         Severity: severity,
                         Source: source,
                         Exceptions: response.Exceptions.Exception
                     });
+                    if (message) {
+                        logger.error(message);
```

Comment:
Should be `logger.info` for `severity === "Info"` and logger.error for `severity === "Error"`